src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -717,6 +717,283 @@ For example, if the page that was copied from is called `Leads`, then the newly
+====
+
+In the `Upload icon` sub-section you can drag and drop or click on the button and choose an icon.
+In the `Or add link to icon` sub-section you can add a link to an icon and preview it.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-icon.png",
+    "alias": "page-edit-configure-icon-preview",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 570,
+    "y": 40,
+    "w": 780,
+    "h": 330
+  },
+  "annotations": [{
+    "x": 1300,
+    "y": 240,
+    "w": 40,
+    "h": 40
+  }]
+}
+----
+
+After uploading an icon a third sub-section will appear.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-uploaded-icon.png",
+    "alias": "page-edit-configure-uploaded-icon-section",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 558,
+    "y": 35,
+    "w": 800,
+    "h": 570
+  },
+  "annotations": [{
+    "x": 580,
+    "y": 300,
+    "w": 760,
+    "h": 180
+  }]
+}
+----
+
+[NOTE]
+====
+After uploading an icon, it will be automatically selected
+====
+
+In it you can select 
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-selected-icon.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 555,
+    "y": 43,
+    "w": 796,
+    "h": 575
+  }
+}
+----
+
+or unselect an icon.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-uploaded-icon.png",
+    "alias": "page-edit-configure-uploaded-icon-unselected",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 555,
+    "y": 43,
+    "w": 796,
+    "h": 575
+  }
+}
+----
+
+If an icon is selected and btn:[Save and close] is clicked, the selected icon will be applied in the `Page icon` section.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-set-icon.png",
+    "alias": "page-edit-configure-set-icon-focus-page-content",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 555,
+    "y": 150,
+    "w": 1100,
+    "h": 685
+  },
+  "annotations": [{
+    "x": 585,
+    "y": 440,
+    "w": 110,
+    "h": 105
+  }]
+}
+----
+
+After clicking on the btn:[Save] button, the icon will be shown in the sidebar next to the page name.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-edit-configure-set-icon.png",
+    "alias": "page-edit-configure-set-icon-focus-sidebar-content",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 0,
+    "y": 0,
+    "w": 750,
+    "h": 700
+  },
+  "annotations": [{
+    "x": 30,
+    "y": 315,
+    "w": 75,
+    "h": 35
+  }]
+}
+----
+
+[IMPORTANT]
+====
+After the page config has been set, the icon will be set/updated in the sidebar
+====
```

Comment:
How is this important? Also, wasn't this already mentioned in the above paragraph?